dependencies/build_imagemagick.sh

```diff
@@ -6,14 +6,14 @@ set -eu
 
 echo Compiler: ${DEB_HOST_MULTIARCH} Arch: ${DEB_HOST_ARCH}
 
-apt-get install -y libjxl-dev:${DEB_HOST_ARCH} liblcms2-dev:${DEB_HOST_ARCH} liblqr-1-0-dev:${DEB_HOST_ARCH} libdjvulibre-dev:${DEB_HOST_ARCH} libjpeg62-turbo-dev:${DEB_HOST_ARCH} libopenjp2-7-dev:${DEB_HOST_ARCH} libopenexr-dev:${DEB_HOST_ARCH} libpng-dev:${DEB_HOST_ARCH} libtiff-dev:${DEB_HOST_ARCH} libwebp-dev:${DEB_HOST_ARCH} libxml2-dev:${DEB_HOST_ARCH} libfftw3-dev:${DEB_HOST_ARCH} zlib1g-dev:${DEB_HOST_ARCH} liblzma-dev:${DEB_HOST_ARCH} libbz2-dev:${DEB_HOST_ARCH} libexif-dev:${DEB_HOST_ARCH} libwmf-dev:${DEB_HOST_ARCH}
+apt-get install -y libjxl-dev:${DEB_HOST_ARCH} liblcms2-dev:${DEB_HOST_ARCH} liblqr-1-0-dev:${DEB_HOST_ARCH} libdjvulibre-dev:${DEB_HOST_ARCH} libjpeg62-turbo-dev:${DEB_HOST_ARCH} libopenjp2-7-dev:${DEB_HOST_ARCH} libopenexr-dev:${DEB_HOST_ARCH} libpng-dev:${DEB_HOST_ARCH} libtiff-dev:${DEB_HOST_ARCH} libwebp-dev:${DEB_HOST_ARCH} libxml2-dev:${DEB_HOST_ARCH} libfftw3-dev:${DEB_HOST_ARCH} zlib1g-dev:${DEB_HOST_ARCH} liblzma-dev:${DEB_HOST_ARCH} libbz2-dev:${DEB_HOST_ARCH}
 URL=$(curl -s https://api.github.com/repos/ImageMagick/ImageMagick/releases/latest | grep "tarball_url" | cut -d : -f 2,3 | tr -d ' ,"')
 echo download ImageMagick from $URL
 curl -L -o ./magick.tar.gz "$URL"
 
 tar xfv ./magick.tar.gz
 cd ImageMagick-*
-./configure --enable-static --enable-delegate-build --disable-shared --with-x=no --host=${DEB_HOST_MULTIARCH}
+./configure --enable-64bit-channel-masks --enable-static --enable-delegate-build --disable-shared --with-x=no --with-magick-plus-plus=no --host=${DEB_HOST_MULTIARCH}
```

Comment:
Could you please explain these parameters and their impact? Did you review all possible parameters and set the optimal ones?

Reply:
It enables ImageMagcik working with more channels, see https://imagemagick.org/script/multispectral-imagery.php.

Reply:
Thanks. What about other parameters? Are there some other useful ones, which we could add or modify to optimize and enhance ImageMagick for our use case?

Reply:
I didn't find other parameters relating to image format. I believe this 64bit channel masks is not a popular feature, but it doesn't increase the binary size.